source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -114,9 +114,12 @@
     }
 
     EvaluationViewModel.prototype.isValueEditable = function (calification) {
-        return !this.evaluation.finished && ((((this.userLogged == calification.calificationColumn.evaluatorEmployee) ||
-            (this.userView == 3 && calification.calificationColumn.evaluatorEmployee == "_company")) && !calification.calificationColumn.finished) ||
-            this.evaluation.readyForDevolution && this.userView == 3 && (calification.calificationColumn.owner == 3 || calification.calificationColumn.owner == 0))
+        return !this.evaluation.finished
```

Comment:
This whole function is hard to read (understand) unless you have a good context. Maybe as a code improvement (not needed to merge this PR) we could move some of the IF statements to new methods. For example: 

``` javascript
function IsEvaluationFinished()
{
    return this.evaluation.finished;
}

...
return !IsEvaluationFinished() && ((IsCompanyEvaluationFinished() || ...))
...

```
This example is simple, but for the more complex ones it might be more readable.